web/app.go

```diff
@@ -170,7 +170,8 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 	webEngine.GET("/hosts/:name", NewHostHandler(deps.consul, deps.subscriptionsService))
 	webEngine.GET("/catalog", NewChecksCatalogHandler(deps.checksService))
 	webEngine.GET("/clusters", NewClusterListHandler(deps.clustersService))
-	webEngine.GET("/clusters/:id", NewClusterHandler(deps.consul, deps.checksService))
+	webEngine.GET("/clusters/:id", NewClusterHandler(deps.clustersService))
+	webEngine.GET("/clusters/:id/generic", NewClusterGenericHandler(deps.consul))
```

Comment:
I understand that this is a temporary solution to split the consul based handlers with the new ones, right?
Even though it could be still useful once we remove consul to avoid having the `generic` suffix in the URLs in the templates, and manage the logic like this

Reply:
@arbulu89 yes this is temporary, it will be the same URL as before, i just wanted to split the two views because it was easier in the process